service.go

```diff
@@ -116,9 +116,9 @@ type NetworkPoint struct {
 	LastMiss                  time.Time         `json:"last_miss"`
 }
 
-type networkPointWithAddress NetworkPoint
+type networkPointAlt NetworkPoint
```

Comment:
Add a comment what this data type means or why it's needed?

Reply:
Sometimes Tezos uses lame array-like encoding for some objects like [hash/id, object]. Alternative type is needed because it bears different JSON unmarshaller.